src/client/styles/components/_place-card.scss

```diff
@@ -0,0 +1,136 @@
+.place-card {
+    display: flex;
+    background-color: rgba(39,39,39,1);
+    height: 125px;
+    width: 100%;
+    align-content: center;
+    align-items: center;
+}
+
+.place-img-container {
+    display: block;
+    left: 5%;
+    width: 125px;
+    height: 125px;
+    text-align: center;
+    align-content: center;
+    align-items: center;
+    border-radius: 50%;
+    -moz-border-radius: 50%;
+    -webkit-border-radius: 50%;
+    float: left;
+}
+.place-img-container>img {
+    max-height: 80%;
+    max-width: 80%;
+    border-radius: 50%;
+    text-align: center;
+    align-content: center;
+    align-items: center;
+}
+
+.place-details {
+    float: right;
+    display:flex;
+    flex-direction: column;
+    position: absolute;
+    left: 150px;
```

Comment:
using px could result in issues with mobile responsiveness. can you please check if it looks fine on mobile and desktop with different screen sizes?

Reply:
This left-alignment is fixed because the image size on this div's left is fixed at 125x125